lib/docx/body-reader.js

```diff
@@ -133,14 +134,27 @@ function BodyReader(options) {
             return readXmlElements(element.children)
                 .map(function(children) {
                     var properties = _.find(children, isRunProperties);
-                    
-                    return new documents.Run(
-                        children.filter(negate(isRunProperties)),
-                        properties
-                    );
+                    var filteredChildren = children.filter(negate(isRunProperties));
+
+                    if (properties && properties.styleName === 'Hyperlink' && lastInstrText) {
```

Comment:
is `styleId` better to use for this sort of logic? Essentially the only way to tell if the text contained in a run is a hyperlink (from what I can tell based on the spec and from playing around in Word) is to have a `w:rStyle` tag with the value of `Hyperlink`.